### PR TITLE
Switched to static reddit OAuth for local

### PR DESCRIPTION
**Pull request by rhysyngsun**
#### What are the relevant tickets?
Prerequisite to #279, sister to mitodl/reddit-config#14

#### What's this PR do?
This PR fixes the `OPEN_DISCUSSIONS_REDDIT_CLIENT_ID` and `OPEN_DISCUSSIONS_REDDIT_SECRET` values. 

#### How should this be manually tested?
Clear the two keys above from your `.env`.

Test in tandem with the sibling PR, verify you can still make calls using `channels.api.Api` (one is sufficient since it tests both).

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -38,10 +38,12 @@ services:
       OPEN_DISCUSSIONS_DB_DISABLE_SSL: 'True'
       OPEN_DISCUSSIONS_REDDIT_URL: https://reddit.local
       OPEN_DISCUSSIONS_REDDIT_VALIDATE_SSL: 'False'
+      OPEN_DISCUSSIONS_REDDIT_CLIENT_ID: od_client_id
+      OPEN_DISCUSSIONS_REDDIT_SECRET: od_client_secret
       ELASTICSEARCH_URL: elastic:9200
       CELERY_TASK_ALWAYS_EAGER: 'False'
       CELERY_BROKER_URL: redis://redis:6379/4
-      CELERY_RESULT_BACKEND: redis://redis:6379/4
+      CELERY_RESULT _BACKEND: redis://redis:6379/4
```
Comment (noisecapella) [inline docker-compose.yml]: This looks like a typo

Comment (rhysyngsun) [reply docker-compose.yml]: eek, fixed!

File: README.md [hunk 2]
```diff
@@ -4,36 +4,13 @@ This provides a discussion forum for use with other MIT applications.
...
-    
-        ```
-        [secrets]
-        generate_refresh_token_client_id = <base64 encoded value here>
-        ```
- 
- 1. cd to `reddit-config` and reload/re-provision the Vagrant VM to update the config with your client ID: 
-    `vagrant reload && vagrant provision`.
- 1. Run the open_discussions container and navigate to the running site in your browser 
+ 1. Run the open_discussions container and navigate to the running site in your browser
```
Comment (noisecapella) [inline README.md]: Can you change `open_discussions container` to `the containers for open-discussions`?

**Comment by rhysyngsun** [comment]
Wording fixed

**Pull request merged**